.github/workflows/package-publish.yml

```diff
@@ -0,0 +1,81 @@
+name: Package build and publish
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version'
+        required: true
+        type: string
+      npm_tag:
+        description: 'release tag'
+        required: false
+        type: string
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          submodules: 'recursive'
+          fetch-depth: 0
+      - name: Enable corepack
+        run: corepack enable
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18.x
+          cache: 'yarn'
+      - name: Check if the release branch exists
+        run: |
+          set -x
+          branch_name="release/v${{ github.event.inputs.version }}"
+          if ! git ls-remote --exit-code --heads origin "$branch_name" > /dev/null; then
+            echo "Branch $branch_name does not exist. Make sure to create the branch and create a Jira ticket with pr-comment-bot."
+            exit 1
+          fi
+      - name: Setup jq
+        if: ${{ github.event.inputs.npm_tag }}
+        uses: dcarbone/install-jq-action@v2
+      - name: Update version in package.json if npm_tag is provided
+        if: ${{ github.event.inputs.npm_tag }}
+        run: |
+          npm_version="${{ github.event.inputs.version }}-${{ github.event.inputs.npm_tag }}"
+          jq --arg npm_version "$npm_version" '.version = $npm_version' package.json > package.json.tmp && mv package.json.tmp package.json
+      - name: Set environments
+        run: |
+          git config --global user.name "sendbird-sdk-deployment"
+          git config --global user.email "sha.sdk_deployment@sendbird.com"
+      - name: Install and Build
+        run: |
+          yarn install --immutable --immutable-cache
+          touch .env.production
+          echo "VITE_CHAT_AI_WIDGET_KEY=${{ secrets.chat_ai_widget_key }}" >> .env.production && \
```

Comment:
```suggestion
```

Reply:
Oh sorry it's my bad.  A commit is missing since I created this PR. Just pushed it. https://github.com/sendbird/sendbird-uikit-react/pull/1099/commits/e71179cba1e2d2851ff95a25b40ff72740d6da02#diff-771047455c9ee791192f052e9028c3c0b1acc9f00e6d8cc38dfe2b15e6c13fe8L54